Add Data Folder FAQ (#303)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

Please see [this great guide](https://github.com/6102bitcoin/FAQ/blob/master/seed.md#3-Storing-your-Seed) on how to properly store your seed.
::::

::::details

> ### Where is the Wasabi data folder?

Reviewer: ```suggestion
### Where can I find the Wasabi data folder?
```

> * Windows: `/Users/{your username}/AppData/Roaming/WalletWasabi/client`

Reviewer 2: ```suggestion
* Windows: `/Users/{your username}/AppData/Roaming/WalletWasabi/Client`
```

* Linux: `/Home/.walletwasabi/client`

> * MacOS: `/Users/{your username}/.walletwasabi/client`

Reviewer 2: I am not sure about this but is the capitalization correct?

Author: Not sure either.

Reviewer 3: yes, David confirmed, this is correct.

:::tip

> You need to mark the “show hidden files” setting to see it

Reviewer 2: ```suggestion
You need to mark the “show hidden files” setting to see it.
```

:::

> You can also easily reach it from inside Wasabi: `File > Open > Data Folder` ::::

Reviewer 2: ```suggestion
You can also easily reach it from inside Wasabi: `File > Open > Data Folder`.
::::
```

::::details
### How do I backup my wallet file?

Although you can backup your private keys with the mnemonic words and password, this is only a last resort recovery.
If you want to also secure your address labels, the anonset and additional metadata, then you can do a digital backup.

> Simply copy the `WalletBackups` folder with the `wallet.json` files from your [Wasabi data folder](/FAQ/FAQ-UseWasabi.md#Where-is-the-Wasabi-data-folder) onto suitable hardware, for example an encrypted USB stick.

Reviewer: ```suggestion
Simply copy the `WalletBackups` folder with the `wallet.json` files from your [Wasabi data folder](/FAQ/FAQ-UseWasabi.md#where-can-i-find-the-wasabi-data-folder) onto suitable hardware, for example an encrypted USB stick.
```

### I forgot my lockscreen PIN, what should I do?

As described in the settings, you can just delete it.

> Open the `UiConfig.json` file inside your [Wasabi data folder](/FAQ/FAQ-UseWasabi.md#Where-is-the-Wasabi-data-folder) and set these entries as follows:

Reviewer: ```suggestion
Open the `UiConfig.json` file inside your [Wasabi data folder](/FAQ/FAQ-UseWasabi.md#where-can-i-find-the-wasabi-data-folder) and set these entries as follows:
```

To increase the number of freshly new generated addresses, you have to increase the `MinGapLimit` json property of your `wallet.json` file.

> * Go to `File/Open/Wallets Folder` or navigate into the `Wallets` folder inside your [Wasabi data folder](/FAQ/FAQ-UseWasabi.md#Where-is-the-Wasabi-data-folder) and open your wallet file.

Reviewer: ```suggestion
* Go to `File/Open/Wallets Folder` or navigate into the `Wallets` folder inside your [Wasabi data folder](/FAQ/FAQ-UseWasabi.md#where-can-i-find-the-wasabi-data-folder) and open your wallet file.
```

### Can I export a list of transactions?

There is currently no convenient way to export a list with transaction details.

> However, you can see the `wallet.json` files inside the `WalletBackups` folder (you can find it in your [Wasabi data folder](/FAQ/FAQ-UseWasabi.md#Where-is-the-Wasabi-data-folder)) which contains all the public keys, labels and anonset.

Reviewer: ```suggestion
However, you can see the `wallet.json` files inside the `WalletBackups` folder (you can find it in your [Wasabi data folder](/FAQ/FAQ-UseWasabi.md#where-can-i-find-the-wasabi-data-folder)) which contains all the public keys, labels and anonset.
```

### Where can I find the logs?

In the top left menu `File > Open` you can see there are several logs available.

> * The `Log File` shows you the general information about Wasabi wallet.

Reviewer 2: ```suggestion
* The `Log File` shows you the general information about Wasabi Wallet.
```

* The `Tor Log File` shows the Tor specific logs.

![](/MenuFileOpen.png)

> Alternatively, you can find the logs inside your [Wasabi data folder](/FAQ/FAQ-UseWasabi.md#Where-is-the-Wasabi-data-folder) :::

Reviewer: ```suggestion
Alternatively, you can find the logs inside your [Wasabi data folder](/FAQ/FAQ-UseWasabi.md#where-can-i-find-the-wasabi-data-folder) :::
```
